SmartcarAuth/SmartcarAuth.swift

```diff
@@ -45,15 +47,17 @@ Smartcar Authentication SDK for iOS written in Swift 3.
         - clientId: app client id
         - redirectUri: app redirect uri
         - scope: app oauth scope
-        - development: optional, shows the mock OEM for testing, defaults to false
+        - development: shows the mock OEM for testing, defaults to false. This is deprecated and has been replaced with testMode.
+        - testMode: optional, launch the Smartcar auth flow in test mode, defaults to nil.
         - completion: callback function called upon the completion of the OAuth flow with the error, the auth code, and the state string
     */
-    @objc public init(clientId: String, redirectUri: String, scope: [String] = [], development: Bool = false, completion: @escaping (Error?, String?, String?) -> Any?) {
+    @objc public init(clientId: String, redirectUri: String, scope: [String] = [], development: Bool =  false, testMode: NSNumber? = nil, completion: @escaping (Error?, String?, String?) -> Any?) {
```

Comment:
Should we set `development` to optional too? Or else you would potentially have both `development` and `testMode` set and it is impossible to just set `testMode`. It would still be backwards compatible @gurpreetatwal

Reply:
ack. see changes

Reply:
I think it is fine to leave it as default to `false`, we made it optional in python-sdk so that we could print a warning. If we can do the equivalent in iOS then I think we should make it nil, otherwise just keep it simpler and leave it default `false`

Reply:
Ok sounds good. I reverted the previous commit.